src/components/navigation/test/Tab-test.js

```diff
@@ -0,0 +1,45 @@
+import { mount } from 'enzyme';
+
+import { testPresentationalComponent } from '../../test/common-tests';
+
+import Tab from '../Tab';
+
+const contentFn = (Component, props = {}) => {
+  return mount(
+    <div role="tablist">
+      <Component {...props}>This is child content</Component>
+    </div>
+  );
+};
```

Comment:
Here and in the tests for `TabList`: our automated common and accessibility tests are working well! Accessibility tests will fail if `Tab` isn't inside of a `tablist`, and `TabList` tests will fail if the `TabList` doesn't have any `tab` children. Sweet.